simphony/testing/abc_check_engine.py

```diff
@@ -292,68 +256,15 @@ def create_dataset_items(self):
                 Point((1.1*i, 2.2*i, 3.3*i), uid=uuid.uuid4()))
         return items
 
-    def check_emtpy_dataset(self, ds):
-        """ Check if a dataset is empty
-        """
 
-        self.assertEqual(0, len(list(p for p in ds.iter_points())))
-        self.assertEqual(0, len(list(e for e in ds.iter_edges())))
-        self.assertEqual(0, len(list(f for f in ds.iter_faces())))
-        self.assertEqual(0, len(list(c for c in ds.iter_cells())))
+class LatticeCudsCheck(CheckEngine):
```

Comment:
This is a suggestion:  

Could `LatticeCudsCheck`, `ParticlesCudsCheck` and `MeshCudsCheck` be renamed to reflect that they are checking how an engine supports that type of DataSet.    What about something like "LatticeEngineCheck" or "EngineLatticeCheck"?